DigitalLearningSolutions.Web/Views/SuperAdmin/Faqs/SuperAdminFaqs.cshtml

```diff
@@ -0,0 +1,44 @@
+@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Faqs
+@using Microsoft.Extensions.Configuration
```

Comment:
Don't think we are using any config values here, so the set up for it can be removed.